CO2_Gadget_EINK.h

```diff
@@ -6,7 +6,7 @@
 #define CO2_Gadget_EINK_h
 
 #ifdef SUPPORT_EINK
-// #define DEBUG_EINK
+#define DEBUG_EINK
```

Comment:
This has moved to platformio.ini
https://github.com/melkati/CO2-Gadget/blob/3006bb031491a0e0ac80d978fc99ff597e1466d6/platformio.ini#L29

Reply:
No problem. It can be moved to platformio.ini
I remove this line from here.